### ISSUE TITLE
Platform for discussing penrose

### ISSUE THREAD

**Issue by albheim** [sminez/penrose#254]
In #26 it was briefly mentioned that maybe it would be nice to have a place to discuss things related to penrose.
I have been thinking the same thing so I thought I would create the requested issue to help gauge the interest.

If you think it is a good idea to have somewhere to discuss and ask questions other than this issue tracker, I guess like this post. 

If you have a suggestion of what platform you would like, add a reply about it. If someone already replied with that platform, then like the existing reply instead.

**Comment by albheim** [comment]
Github discussions

**Comment by albheim** [comment]
Discord

**Comment by sminez** [comment]
A while ago I tentatively set up a discord server for this purpose but never made an announcement for it. I've used discord for a variety of things before but never tried GitHub discussions so that would be my preference if I'm honest.

I'd be interested to see what level of interest there is for a forum of this nature if I'm honest! Other than stars on GitHub I'm not actually sure how many people are making use of Penrose as a daily driver so I've got no idea if it'd be a load of people wanting to discuss things or a handful 😅

**Comment by albheim** [comment]
The benefit of discord is then that @sminez already have it and is comfortable with that, I think personally I'm also more used to it. 

The benefit of github discussions is that it is already available to people using github, so no extra account needed if you are already on here. I think it is just flip a switch in the settings of the repo and it should turn up as an extra tab along with the PRs, Actions, Wiki...

Personally I don't use penrose as a daily driver, have never fully taken the leap to a tiling wm, but I'm interested and I like playing around with them so might build up towards making a switch. And penrose seems like an interesting alternative then for many reasons. I would probably use a chat/forum more for asking for help with simple things rather than discussing interesting development issues, at least in the start :)

Btw, I don't know why the enhancement label was added, maybe it was based on me picking some specific type of issue? Doesn't really matter to me, just noticed it now and was surprised it said I added it...

**Comment by JohnDowson** [comment]
I think a big benefit of GH discussions is that they are indexed by search engines.

**Comment by alextreichler** [comment]
I am not sure if this is the best place, but I do think some place to ask questions might be good. I just started looking at Penrose (I am currently using xmonad). I followed the getting started, set everything up… logged out and logged in using Penrose, but then it freezes my computer.  I can still move my mouse, but everything else is frozen. I have to manually restart my computer.  This is on arch.  There are no details I. The Penrose.log file. Having a place to ask some ideas on getting this resolved would be helpful. Thanks.
